build/metadata/github-actions/README.md

```diff
@@ -87,3 +87,37 @@ If the action fails, there may be a problem with your configuration. To investig
 | projKey | Key of the LaunchDarkly project associated with this repository. Found under Account Settings -> Projects in the LaunchDarkly dashboard. Cannot be combined with `projects` block in configuration file. | `false` |  |
 | repoName | The repository name. Defaults to the current GitHub repository. | `false` |  |
 <!-- action-docs-inputs -->
+
+# Additional inputs
+
+All [command line flags](https://github.com/launchdarkly/ld-find-code-refs/blob/main/docs/CONFIGURATION.md#command-line) are available as environment variables following the "upper snake case" format, with a prefix of `LD_`. For example, the command line option `dir` may be set as an environment variable.
+
+For example, if your action checks out multiple repositories you may override the default `dir` which is `GITHUB_WORKSPACE`.
+
+> GITHUB_WORKSPACE - The default working directory on the runner for steps, and the default location of your repository when using the checkout action. For example, /home/runner/work/my-repo-name/my-repo-name.
```

Comment:
```suggestion
The default `dir` is `GITHUB_WORKSPACE`. This is the default working directory on the runner for steps and the default location of your repository when using the checkout action, for example, `/home/runner/work/my-repo-name/my-repo-name`.

If your action checks out multiple repositories, override the default `dir`.
```
If I'm understanding this correctly, there are two ideas here -- describing the default and when you need to override it. Typically the default description goes first, so that readers learn more about the option before they learn when/how to override.